### PR TITLE
[SPARK-41479][K8S][DOCS] Add `IPv4 and IPv6` section to K8s document

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
### What changes were proposed in this pull request?

This PR aims to add `IPv4 and IPv6` section to K8s document.

### Why are the changes needed?

To document IPv6-only environment more explicitly.

### Does this PR introduce _any_ user-facing change?

No. This is a documentation PR.

### How was this patch tested?

Manually review.

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
cc @viirya

File: docs/running-on-kubernetes.md [hunk 1]
```diff
@@ -1418,15 +1438,17 @@ See the [configuration page](configuration.html) for information on Spark config
...
   </td>
   <td>3.4.0</td>
 </tr>
 <tr>
   <td><code>spark.kubernetes.driver.service.ipFamilies</code></td>
   <td><code>IPv4</code></td>
   <td>
-    A list of IP families for K8s Driver Service.
+    A list of IP families for K8s Driver Service. Valid values are
+    <code>IPv4</code> and <code>IPv6</code>.
```
Comment (viirya) [inline docs/running-on-kubernetes.md:1451]: Is `IPv4,IPv6` also valid value?

**Review by viirya — APPROVED** [review]
Looks good to me. Thanks @dongjoon-hyun

File: docs/running-on-kubernetes.md (hunk 1 above)
Comment (viirya) [reply docs/running-on-kubernetes.md:1451]: Oh, this is a list config?

Comment (dongjoon-hyun) [reply docs/running-on-kubernetes.md:1451]: Yes~

**Comment by dongjoon-hyun** [comment]
Merged to master for Apache Spark 3.4. Thank you, @viirya .

**Pull request closed without merging**